src/bin/pgcopydb/ld_stream.h

```diff
@@ -135,32 +135,34 @@ typedef struct LogicalMessageTupleArray
 	LogicalMessageTuple *array; /* malloc'ed area */
 } LogicalMessageTupleArray;
 
+typedef struct LogicalMessageRelation
+{
+	char *nspname;
+	char *relname;
```

Comment:
Add inline comment that this is a malloc'ed area, please, as we do in the other cases.